### DISCUSSION TITLE
Welcome to AkashaScanner Discussions!

### DISCUSSION THREAD

**Discussion by xenesty** [xenesty/AkashaScanner#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kimoi4Weeb** [comment]
Hi so there is no .exe file that i can run but only a .csproj file present. So could you tell me how to fix this.
Thanks 
![image_2022-12-15_190838082](https://user-images.githubusercontent.com/120654430/207873119-e4cad156-f859-4c05-96db-0c46475a4390.png)
![image](https://user-images.githubusercontent.com/120654430/207873195-a6897781-b3cf-4893-9057-2d90e715ff58.png)

**Reply by xenesty** [reply]
You should download the release, not the source code.
